cfgov/jinja2/v1/_includes/organisms/snippet-list.html

```diff
@@ -55,7 +55,7 @@
     } ) }}
 
     {% set snippets = get_snippets(value.snippet_type) %}
-    {% set filtered_snippets = snippets.objects.filter_by_tags(value.tags) %}
+    {% set filtered_snippets = snippets.objects.filter_by_tags(value.tags).order_by('order', 'title') %}
```

Comment:
One thought: instead of putting this into the SnippetList template, you could attach this to the model as its default ordering, e.g.

```py
class Resource(ClusterableModel):
    ...

    class Meta:
        ordering = ('order', 'title')
```

This would make any queries to this model (like `Resource.objects.all()`, `Resource.objects.filter(...)`) automatically use this ordering by default. It'd also solve your problem with `order` not existing on the `Contact` snippet, because you wouldn't need to add it there.